reconcile/external_resources/manager.py

```diff
@@ -374,7 +383,9 @@ def handle_dry_run_resources(self) -> None:
             if (
                 r.action == Action.APPLY
                 and state.reconciliation.resource_hash != r.resource_hash
-            ) or r.action == Action.DESTROY:
+            ) or (
+                r.action == Action.DESTROY and not state.resource_status.is_in_progress
```

Comment:
This avoids running the destroy reconciliation in DRY-RUN mode in the Erv2 related PRs